src/components/Footer/Office.js

```diff
@@ -19,7 +19,9 @@ const Office = ({ name, telephone, email, streetAddress }) => (
       <StyledBodyPrimary itemProp="telephone">{telephone}</StyledBodyPrimary>
       {email ? (
         <StyledBodyPrimary>
-          <a href={`mailto:${email}`}>{email}</a>
+          <a href={`mailto:${email}`} title={`Email yld ${name} Office`}>
```

Comment:
the only content issue I have quibble with there, is that you'll end up with "Email yld London - HQ Office" which doesn't sound ideal.

Reply:
yes, thought of that, it still makes sense though no?

Reply:
True, it does!